### ISSUE TITLE
Target .NET Standard 2.0

### ISSUE THREAD

**Issue by omatrot** [TrackableEntities/TrackableEntities.Core#35]
TrackableEntities.EF.Core 3.1 is now targeting .NET Standard 2.1 but EF 3.1 targets .NET Standard 2.0.
I can't use it anymore in a .NET 4.7.2 project.

Furthermore, I have a method not found exception using TrackableEntities.EF.Core 1.2.0 (targeting .NET Standard 2.0) and EF Core 3.1.

```
 <Exception>
    <ExceptionTypeName>System.MissingMethodException</ExceptionTypeName>
    <Message>Method not found: 'Void Microsoft.EntityFrameworkCore.ChangeTracking.IEntityEntryGraphIterator.TraverseGraph(Microsoft.EntityFrameworkCore.ChangeTracking.EntityEntryGraphNode, !!0, System.Func`3&lt;Microsoft.EntityFrameworkCore.ChangeTracking.EntityEntryGraphNode,!!0,Boolean&gt;)'.</Message>
    <Source>TrackableEntities.EF.Core</Source>
    <StackTrace>   at TrackableEntities.EF.Core.Internal.DbContextExtensionsInternal.TraverseGraph(DbContext context, Object item, Action`1 callback)
```
Is there a solution to this problem?

**Comment by tonysneed** [comment]
I can release a version of the package targeting NET Standard 2.0.

**Comment by omatrot** [comment]
I'd like that. Thanks.

**Comment by omatrot** [comment]
Hi @tonysneed any news on this? Could I have a release date?

**Comment by tonysneed** [comment]
Working on it tomorrow...

**Comment by tonysneed** [comment]
Fixed by #38

**Comment by tonysneed** [comment]
Released: https://www.nuget.org/packages/TrackableEntities.EF.Core/3.1.1
